edk2toollib/windows/locate_tools.py

```diff
@@ -174,7 +174,11 @@ def QueryVcVariables(keys: list, arch: str = None, product: str = None, vs_versi
     result = {}
     ret, vs_path = FindWithVsWhere(product, vs_version)
     if ret != 0 or vs_path is None:
-        logging.warning("We didn't find VS path or otherwise failed to invoke vsWhere")
+        logging.error("We didn't find VS path or otherwise failed to invoke vsWhere")
```

Comment:
if this is trying to compare against none you should check if "is None".  as the below conditional will also trigger if vs_version == 0

Reply:
now checks via `if vs_version is not None:` as suggested